code/upaya/app/assets/javascripts/tech_support.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
This isn't related to Devise authentication and is an empty file. Can we add it only when we need it?